web/styles/dark_theme.css

```diff
@@ -505,33 +505,15 @@
         }
     }
 
-    .recent_view_participant_overflow {
-        color: hsl(0deg 0% 100%) !important;
-        background-color: hsl(211deg 18% 25%) !important;
+    .btn-default {
+        background-color: hsl(211deg 29% 14%);
```

Comment:
This has been done to avoid using `!important` in `.recent_view_participant_overflow` and `.btn-recent-filters:focus`.

Reply:
Since `btn-default` is a class introduced by bootstrap, this is something we are trying to run away from. Let's try to find a way to fix it without using that class, even if we have to use `!important`.

Reply:
`btn-default` class is removed by this commit 35ba96bd9d49ac2e720028d9fc73f9d568740f97.